src/main/java/com/google/sps/servlets/MenteeSignupServlet.java

```diff
@@ -41,17 +41,12 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     // Set up map to store majors.
     Map<Integer, String> majors = new HashMap<Integer, String>();
     
-    // Set up variables needed to connect to MySQL database.
-    String url = Utility.SQL_LOCAL_URL;
-    String user = Utility.SQL_LOCAL_USER;
-    String password = Utility.SQL_LOCAL_PASSWORD;
-
     // Set up query to retrieve all subject tags.
     String query = "SELECT * FROM Major";
 
     try {
       // Establish connection to MySQL database.
-      Connection connection = DriverManager.getConnection(url, user, password);
+      Connection connection = DriverManager.getConnection(Utility.SQL_LOCAL_URL, Utility.SQL_LOCAL_USER, Utility.SQL_LOCAL_PASSWORD);
```

Comment:
Probably wrapping this line will improve the readability.

Reply:
Hadn't noticed it was above the 100-column limit. Done!